classes/requests/class-dintero-checkout-request.php

```diff
@@ -225,11 +225,13 @@ protected function process_response( $response, $request_args, $request_url ) {
 	protected function log_response( $response, $request_args, $request_url ) {
 		$body = json_decode( $response['body'], true );
 
-		$method   = $this->method;
-		$title    = $this->log_title;
-		$code     = wp_remote_retrieve_response_code( $response );
-		$order_id = $body['id'] ?? json_decode( $request_args['body'], true )['id'] ?? null;
-		$log      = Dintero_Checkout_Logger::format_log( $order_id, $method, $title, $request_args, $response, $code, $request_url );
+		$method        = $this->method;
+		$title         = $this->log_title;
+		$code          = wp_remote_retrieve_response_code( $response );
+		$order_id      = $body['id'] ?? json_decode( $request_args['body'], true )['id'] ?? null;
+		$checkout_flow = $this->settings['checkout_flow'] ?? 'express_popout';
```

Comment:
It would be better if we logged `N/A` or something similar instead of assuming `express_popout` as that could be misleading.

Reply:
But the[ default for the setting](https://github.com/Dintero/Dintero.Checkout.WooCommerce.V2/blob/develop-subscription/classes/class-dintero-checkout-settings-fields.php#L102) is `express_popout` right? So if its not set, that is what should be the setting value used in other places too if I'm not misstaken?

Reply:
That was my reason behind choosing express_popout as fallback. Let me know what you think would be best to use :)